data/address.go

```diff
@@ -51,6 +57,18 @@ func (a *address) IsValid() bool {
 	return len(a.bytes) == core.AddressBytesLen
 }
 
+// Pretty returns a short version of the bech32 address
+func (a *address) Pretty() string {
+	bech32Addr := a.AddressAsBech32String()
+	if len(bech32Addr) <= offsetPretty {
```

Comment:
```suggestion
	if len(bech32Addr) * 2 <= offsetPretty {
```

the offset must be less or equal with half of len(bech32Addr)

Reply:
thanks, fixed